### ISSUE TITLE
nanmean warnings

### ISSUE THREAD

**Issue by eculler** [earthlab-education/ea-lidar-uncertainty-review#49]
@eageo this is in reference to PR #29 

There are numerous warnings in your Jupyter notebook:
```
<ipython-input-8-d0ab17fd4069>:60: RuntimeWarning: Mean of empty slice
  mean_ndvi = np.nanmean(ndvi_clean)
```
Perhaps related to using np to take the mean of an xr DataArray?

**Comment by esgeo** [comment]
I noted this when I ran the code. I will try to sort out another way to get the mean ndvi. Thanks

**Comment by esgeo** [comment]
@eculler I have kind of hit a wall with this and have just chosen to suppress the warnings in the `np.nanmean` block of code

From my research on this topic, I have found the mean of smpty slice warning to be a common issue with no actual consequence while using `np.nanmean` (see issue and stack overflow links below) in which much of the online advice (albeit 3-4 years old) is to simply ignore the warnings. This all seems way beyond my comfort level, but it appears it has something to do with the imbedded `dask.array` function that xarray uses. 

https://github.com/dask/dask/issues/3245#issue-302182457


As suggested in this thread below, I used the code 

`with warnings.catch_warnings():
    warnings.simplefilter("ignore", category=RuntimeWarning)`

to suppress warnings for the block that runs the mean of empty slices
 
https://stackoverflow.com/questions/29688168/mean-nanmean-and-warning-mean-of-empty-slice

Hope that is acceptable.
